<?xml version="1.0"?>

<launch>
   <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor.gazebo.xacro" />

   <group ns="uav1">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav1" />
       <arg name="tf_prefix" value="uav1" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-6.0" />
       <arg name="y" value="0.0" />
     </include>
   </group>

   <group ns="uav2">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav2" />
       <arg name="tf_prefix" value="uav2" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="6.0" />
       <arg name="y" value="0" />
     </include>
   </group>
   <group ns="uav3">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav3" />
       <arg name="tf_prefix" value="uav3" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="0.0" />
       <arg name="y" value="6.0" />
     </include>
   </group>
   <group ns="uav4">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav4" />
       <arg name="tf_prefix" value="uav4" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="6" />
       <arg name="y" value="6.0" />
     </include>
   </group>
   <group ns="uav5">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav5" />
       <arg name="tf_prefix" value="uav5" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-6" />
       <arg name="y" value="6" />
     </include>
   </group>
   <group ns="uav6">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav6" />
       <arg name="tf_prefix" value="uav6" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="0" />
       <arg name="y" value="-6" />
     </include>
   </group>
   <group ns="uav7">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav7" />
       <arg name="tf_prefix" value="uav7" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="6.0" />
       <arg name="y" value="-6.0" />
     </include>
   </group>
   <group ns="uav8">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav8" />
       <arg name="tf_prefix" value="uav8" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-6.0" />
       <arg name="y" value="-6" />
     </include>
   </group>
   <!-- <group ns="uav9">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav9" />
       <arg name="tf_prefix" value="uav9" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="0.0" />
       <arg name="y" value="0.0" />
     </include>
   </group> -->
   <!-- <group ns="uav10">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav10" />
       <arg name="tf_prefix" value="uav10" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="6.0" />
       <arg name="y" value="12" />
     </include>
   </group>
   <group ns="uav11">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav11" />
       <arg name="tf_prefix" value="uav11" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-6.0" />
       <arg name="y" value="0" />
     </include>
   </group>
   <group ns="uav12">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav12" />
       <arg name="tf_prefix" value="uav12" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="0" />
       <arg name="y" value="0" />
     </include>
   </group>
   <group ns="uav13">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav13" />
       <arg name="tf_prefix" value="uav13" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="6.0" />
       <arg name="y" value="0" />
     </include>
   </group>
   <group ns="uav14">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav14" />
       <arg name="tf_prefix" value="uav14" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-6.0" />
       <arg name="y" value="-12" />
     </include>
   </group>
   <group ns="uav15">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav15" />
       <arg name="tf_prefix" value="uav15" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="6.0" />
       <arg name="y" value="-12" />
     </include>
   </group>
   <group ns="uav16">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav16" />
       <arg name="tf_prefix" value="uav16" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-12.0" />
       <arg name="y" value="6" />
     </include>
   </group>
   <group ns="uav17">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav17" />
       <arg name="tf_prefix" value="uav17" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="12.0" />
       <arg name="y" value="6" />
     </include>
   </group>
   <group ns="uav18">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav18" />
       <arg name="tf_prefix" value="uav18" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-12.0" />
       <arg name="y" value="-6" />
     </include>
   </group>
   <group ns="uav19">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav19" />
       <arg name="tf_prefix" value="uav19" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="12.0" />
       <arg name="y" value="-6" />
     </include>
   </group>
   <group ns="uav20">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav20" />
       <arg name="tf_prefix" value="uav20" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-6.0" />
       <arg name="y" value="6" />
     </include>
   </group>
   <group ns="uav21">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav21" />
       <arg name="tf_prefix" value="uav21" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-18.0" />
       <arg name="y" value="12.0" />
     </include>
   </group>
   <group ns="uav22">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav22" />
       <arg name="tf_prefix" value="uav22" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-18.0" />
       <arg name="y" value="6" />
     </include>
   </group>
   <group ns="uav23">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav23" />
       <arg name="tf_prefix" value="uav23" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-18.0" />
       <arg name="y" value="0.0" />
     </include>
   </group>
   <group ns="uav24">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav24" />
       <arg name="tf_prefix" value="uav24" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-18" />
       <arg name="y" value="-6.0" />
     </include>
   </group>
   <group ns="uav25">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav25" />
       <arg name="tf_prefix" value="uav25" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-18" />
       <arg name="y" value="-12" />
     </include>
   </group>
   <group ns="uav26">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav26" />
       <arg name="tf_prefix" value="uav26" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="18" />
       <arg name="y" value="12.0" />
     </include>
   </group>
   <group ns="uav27">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav27" />
       <arg name="tf_prefix" value="uav27" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="18.0" />
       <arg name="y" value="6.0" />
     </include>
   </group>
   <group ns="uav28">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav28" />
       <arg name="tf_prefix" value="uav28" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="18.0" />
       <arg name="y" value="0" />
     </include>
   </group>
   <group ns="uav29">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav29" />
       <arg name="tf_prefix" value="uav29" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="18.0" />
       <arg name="y" value="-6" />
     </include>
   </group>
   <group ns="uav30">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav30" />
       <arg name="tf_prefix" value="uav30" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="18.0" />
       <arg name="y" value="-12" />
     </include>
   </group> -->
</launch>
